suite-native/module-settings/src/screens/SettingsViewOnly.tsx

```diff
@@ -37,16 +121,41 @@ export const SettingsViewOnly = () => {
                             <IconButton
                                 colorScheme="redElevation0"
                                 iconName="close"
-                                onPress={() => handleViewOnlyChange(device)}
+                                onPress={() => handleDisableViewOnly(device)}
                             />
                         ) : (
-                            <Button onPress={() => handleViewOnlyChange(device)}>
+                            <Button onPress={() => toggleViewOnly(device)}>
                                 <Translation id="moduleSettings.viewOnly.enableButton" />
                             </Button>
                         )}
                     </HStack>
                 </Card>
             ))}
+        </>
+    );
+};
+
+export const SettingsViewOnly = () => {
```

Comment:
There are four components in this file and it is too long. Please split it to multiple files for better readability.

I suggest keeping the `EmptyContent` in this file (since it is relatively small with no extra functionality) and creating a new file `ViewOnlySettingsContent` (for example) and moving the `DevicesContent` and `About` components there.

Reply:
59fad99c60